Dockerfile

```diff
@@ -29,12 +29,32 @@ RUN curl --fail -sSL -O https://storage.googleapis.com/kubernetes-release/releas
     && kubectl completion bash > /etc/bash_completion.d/kubectl.sh
 
 # Install kops
-ENV KOPS_VERSION 1.5.1
+ENV KOPS_VERSION 1.7.1
+ENV KOPS_STATE_STORE s3://undefined
+ENV KOPS_STATE_STORE_REGION us-east-1
+ENV AWS_SDK_LOAD_CONFIG=1
+ENV KOPS_FEATURE_FLAGS=+DrainAndValidateRollingUpdate
+ENV KOPS_MANIFEST=/conf/kops/manifest.yaml
+ENV KOPS_TEMPLATE=/templates/kops/default.yaml
+ENV KOPS_BASE_IMAGE=kope.io/k8s-1.6-debian-jessie-amd64-hvm-ebs-2017-05-02
```

Comment:
Is there a newer base image we can use?

Reply:
Yes, updated

ENV KOPS_BASE_IMAGE=kope.io/k8s-1.7-debian-jessie-amd64-hvm-ebs-2017-07-28

https://github.com/kubernetes/kops/blob/master/channels/stable
https://github.com/kubernetes/kops/blob/master/docs/images.md